docs/EasyCommands/blockHandlers/suspension.md

```diff
@@ -156,4 +156,16 @@ Print "Wheel Friction: " + "My Wheel" ratio
 
 #Set Friction to 50%
 set "My Wheel" ratio to 50
+```
+
+## "Attached" Property
+* Primitive Type: Bool
+* Keywords: ```attached, connected```
+* Inverse Keywords: ```detached, disconnected```
+
+Returns true if there is a wheel attached to the suspension.
+
+```
+if "My Wheel" is attached
```

Comment:
i think you mean if "My Wheel" is detached, or not attached.

Reply:
ups